src/schema/BIDS_VERSION

```diff
@@ -0,0 +1 @@
+v1.7.1-dev
```

Comment:
who / how would produce this file?  It should become part of the release/post-release process of bids-specification or otherwise it just would get outdated and thus reporting an incorrect version to the user!  I wonder if @bids-maintenance folks would allow it being hardcoded here for that reason.  And if they do , http://github.com/bids-standard/bids-specification/blob/HEAD/Release_Protocol.md should reflect the need to adjust this file as well.

Reply:
Yes, the plan is for it to be auto-generated for release, but within the actual git tracking it's equivalent to https://github.com/bids-standard/bids-specification/blob/c80582efbee220b6f16e20ca795d25179bd95285/mkdocs.yml#L1

Reply:
@yarikoptic ok, I added this to the release protocol.